app/javascript/packages/document-capture/components/documents-step.jsx

```diff
@@ -47,7 +46,7 @@ function DocumentsStep({
   const { flowPath } = useContext(UploadContext);
 
   return (
-    <CaptureTroubleshooting>
+    <div>
```

Comment:
Could we get away with using a fragment instead of a `<div>`, to avoid an extra DOM element?

(Requires matching closing tag below)

```suggestion
    <>
```

Reply:
I think you're right! As I said to Eileen:

> To be honest, I'm (fairly) new to React and didn't know about fragments. I just knew that everything in the `return` had to be wrapped by an element and thought `div` was the best option. TIL! Pushing up a change to use a fragment shortly.